### ISSUE TITLE
Rule Proposal: `vue/no-extra-whitespaces` 

### ISSUE THREAD

**Issue by riophae** [vuejs/eslint-plugin-vue#133]
**Please describe what the rule should do:**

This rule warns about the usage of useless spaces.

**What category of rule is this? (place an "X" next to just one item)**

`[x]` Enforces code style
`[ ]` Warns about a potential error
`[ ]` Suggests an alternate way of doing something
`[ ]` Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```html
<template>
  <div class="wrapper">
    <div class="example1" ></div>
    <div class="example2"  @click="handleClick">
      <!-- Notice there are two spaces between `class` and `@click` -->
    </div>
    <div class="example3 ">
      <!-- Is it possible to catch extra spaces in attribute value? -->
    </div>
  </div>
</template>
```

However, the following should be considered fine:

```html
<template>
  <div class="wrapper">
    <self-closing />
    <div 
      class="multiple-line"
      @click="handleClick"
      >
    </div>
  </div>
</template>
```

The suggested rule name is likely not good enough.

**Comment by armano2** [comment]
@riophae everything is possible

i like this rule (i will implement it today) but i don't think that catching extra spaces in attribute value should be handled (at-least not by default)

**Comment by riophae** [comment]
I think it should be no problem as far as we can confirm the attribute value is an expression rather than plain string, such as vue-related attributes, event attributes, `class` and `style`, etc.

**Comment by mysticatea** [comment]
Thank you for this issue.

Sounds good to me.
I think that the rule name is better if it's `vue/no-multi-spaces` or something like. [no-multi-spaces](http://eslint.org/docs/rules/no-multi-spaces) rule exists in core.

**Comment by armano2** [comment]
@mysticatea i like `no-multi-spaces` i will rename it
